rfd/00058-poc/poc.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
```

Comment:
This script can probably be removed now.

Reply:
Removed